upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.7.yaml.template

```diff
@@ -1,412 +1,136 @@
-{{- $etcd_scheme := EtcdScheme }}
-kind: ConfigMap
 apiVersion: v1
```

Comment:
According to https://github.com/kubernetes/kops/pull/7474/files#diff-dec2adb888d136e5fcab51a2af69403dL425 Cilium doesn't support 1.7 so this shouldn't be updated? Also I really don't like forcing old users to this new version, especially when they really shouldn't be using k8s that's this old anyway.

Reply:
I’m a little confused with that statement: people should use sth that old so we shouldn’t force people to update?

Reply:
If a user is still running k8s 1.7 - k8s 1.12, forcing a cilium upgrade of this magnitude is likely aggressive and has a strong likelihood of causing instability.